File: Fuentes/Principal (Renovado)/Entidades/BaseDatos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data.SqlClient;
using System.Configuration;

namespace Entidades
{
   public class BaseDatos
    {

        private string cadenaConexionInformacion;
        private string cadenaConexionCatalogo;
        public static SqlConnection conexionInformacion = new SqlConnection();
        public static SqlConnection conexionCatalogo = new SqlConnection(); 

        public string CadenaConexionInformacion
        {
            get { return cadenaConexionInformacion; }
            set { cadenaConexionInformacion = value; }
        }
        public string CadenaConexionCatalogo
        {
            get { return cadenaConexionCatalogo; }
            set { cadenaConexionCatalogo = value; }
        } 
          //public static SqlConnection conexionPrincipal = new SqlConnection(ConfigurationManager.ConnectionStrings["conexionPrincipal"].ConnectionString);
            
        public void AbrirConexionInformacion()
        {

            this.CadenaConexionInformacion = string.Format("Data Source=SYS21ALIEN03-PC\\SQLEXPRESS;Initial Catalog={0};User Id=AdminBerry;Password=@berry", this.CadenaConexionInformacion);
            conexionInformacion.ConnectionString = this.CadenaConexionInformacion; 

        }

        public void AbrirConexionCatalogo()
        {

            this.cadenaConexionCatalogo = string.Format("Data Source=SYS21ALIEN03-PC\\SQLEXPRESS;Initial Catalog={0};User Id=AdminBerry;Password=@berry", this.cadenaConexionCatalogo);
            conexionCatalogo.ConnectionString = this.cadenaConexionCatalogo; 

        }
 
    }
    
}

File: Fuentes/Reporte de Recepcion/Entidades/BaseDatos.vb
Imports System.Data.SqlClient

Public Module BaseDatos

    Private cadenaConexionConfiguracion As String
    Private cadenaConexionCatalogo As String
    Private cadenaConexionEmpaque As String
    Public conexionConfiguracion As New SqlConnection()
    Public conexionCatalogo As New SqlConnection()
    Public conexionEmpaque As New SqlConnection()

    Public Property ECadenaConexionConfiguracion() As String
        Get
            Return BaseDatos.cadenaConexionConfiguracion
        End Get
        Set(value As String)
            BaseDatos.cadenaConexionConfiguracion = value
        End Set
    End Property
    Public Property ECadenaConexionCatalogo() As String
        Get
            Return BaseDatos.cadenaConexionCatalogo
        End Get
        Set(value As String)
            BaseDatos.cadenaConexionCatalogo = value
        End Set
    End Property
    Public Property ECadenaConexionEmpaque() As String
        Get
            Return BaseDatos.cadenaConexionEmpaque
        End Get
        Set(value As String)
            BaseDatos.cadenaConexionEmpaque = value
        End Set
    End Property

    Public Sub AbrirConexionConfiguracion()

        BaseDatos.ECadenaConexionConfiguracion = String.Format("Data Source={0};Initial Catalog={1};User Id={2};Password={3}", EYELogicaReporteRecepcion.Directorios.instanciaSql, BaseDatos.ECadenaConexionConfiguracion, EYELogicaReporteRecepcion.Directorios.usuarioSql, EYELogicaReporteRecepcion.Directorios.contrasenaSql)
        conexionConfiguracion.ConnectionString = BaseDatos.ECadenaConexionConfiguracion

    End Sub

    Public Sub AbrirConexionCatalogo()

        BaseDatos.ECadenaConexionCatalogo = String.Format("Data Source={0};Initial Catalog={1};User Id={2};Password={3}", EYELogicaReporteRecepcion.Directorios.instanciaSql, BaseDatos.ECadenaConexionCatalogo, EYELogicaReporteRecepcion.Directorios.usuarioSql, EYELogicaReporteRecepcion.Directorios.contrasenaSql)
        conexionCatalogo.ConnectionString = BaseDatos.ECadenaConexionCatalogo

    End Sub

    Public Sub AbrirConexionEmpaque()

        BaseDatos.ECadenaConexionEmpaque = String.Format("Data Source={0};Initial Catalog={1};User Id={2};Password={3}", EYELogicaReporteRecepcion.Directorios.instanciaSql, BaseDatos.ECadenaConexionEmpaque, EYELogicaReporteRecepcion.Directorios.usuarioSql, EYELogicaReporteRecepcion.Directorios.contrasenaSql)
        conexionEmpaque.ConnectionString = BaseDatos.ECadenaConexionEmpaque

    End Sub

End Module
